sys-cluster/pacemaker/pacemaker-2.1.2.ebuild

```diff
@@ -34,39 +34,40 @@ REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
 S="${WORKDIR}/${PN}-${MY_P}"
 
+PATCHES=( "${FILESDIR}"/${P}-command-not-found.patch )
+
 src_prepare() {
 	default
-	sed -i -e "s/ -ggdb//g" configure.ac || die
+	sed -i -e "s/ -ggdb//g" configure.ac || die "Failed to remove default gdb"
 	eautoreconf
 }
 
 src_configure() {
 	# appends lib to localstatedir automatically
-	local myconf=(
-		--with-ocfdir=/usr/$(get_libdir)/ocf
-		--localstatedir=/var
-		--disable-fatal-warnings
-		--disable-static
-		--without-cs-quorum
-		--without-cman
-		--without-heartbeat
-		--with-corosync
-		--with-ais
-		$(use_with acl)
-		$(use_with smtp esmtp)
+	econf \
+		--localstatedir=/var \
+		--disable-fatal-warnings \
+		--disable-static \
+		--without-cs-quorum \
+		--without-cman \
+		--with-heartbeat \
+		--with-corosync \
+		--with-ais \
+		--with-ocfdir=/usr/$(get_libdir)/ocf \
+		$(use_with acl) \
+		$(use_with smtp esmtp) \
 		$(use_with snmp)
-	)
-
-	econf "${myconf[@]}"
 }
 
 src_install() {
 	default
 	python_optimize
 
 	# remove provided initd file as we need support for OpenRC
-	rm -r "${ED}/etc/init.d" || die "Failed to remove old initd"
-	newinitd "${FILESDIR}/${PN}.initd" "${PN}"
+	# Without -f it fails to remove saying "File or directory not found"
```

Comment:
It fails to remove _sometimes_, I think when systemd is installed, as it never installed the init.d in the first place.

If it always failed, we wouldn't bother having the line.

Reply:
What should I have ti do?

Reply:
Just adjust the comment.